### PR TITLE
add simple cluster client for etcd

### PR DESCRIPTION
**Pull request by StupidHod**
@xiang90 @AdoHe waiting for your advice, what should I do about cluster?

### CODE REVIEW

**Comment by AdoHe** [comment]
Generally LGTM. @StupidHod since test issue has been fixed, we need to setup a cluster with more than one node.

**Comment by StupidHod** [comment]
@AdoHe , OK, I will have a test with cluster, try for one more node.

File: src/main/java/com/coreos/jetcd/EtcdClusterImpl.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    /**
+     * removes an existing member from the cluster
+     *
+     * @param memberId the id of the member
+     * @return
+     */
+    @Override
+    public ListenableFuture<MemberRemoveResponse> removeMember(long memberId) {
```
Comment (xiang90) [inline src/main/java/com/coreos/jetcd/EtcdClusterImpl.java]: memberID

File: src/main/java/com/coreos/jetcd/EtcdClusterImpl.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    /**
+     * update peer addresses of the member
+     *
+     * @param memberId the id of member to update
+     * @param endpoints the new endpoints for the member
+     * @return
+     */
+    @Override
+    public ListenableFuture<MemberUpdateResponse> updateMember(long memberId, List<String> endpoints) {
```
Comment (xiang90) [inline src/main/java/com/coreos/jetcd/EtcdClusterImpl.java]: memberID

**Comment by xiang90** [comment]
LGTM

**Pull request merged**